src/fswAlgorithms/effectorInterfaces/thrFiringRemainder/_UnitTest/test_thrFiringRemainder.py

```diff
@@ -19,7 +19,7 @@
 #
 #   Unit Test Script
 #   Module Name:        thrFiringRemainder
-#   Author:             John Alcorn, modified code by Hanspeter Schaub
+#   Author:             John Alcorn, modified code by Hanspeter Schaub and Julian Hammerl
 #   Creation Date:      August 25, 2016
 #
```

Comment:
:small_red_triangle_down: Delete this. The Git history contains all this information

Reply:
Done.